iris_doc/api_tagger.py

```diff
@@ -273,7 +294,43 @@ def _findFunctionScopeStartIndex(self, lines: List[str], startIndex: int) -> int
         return self._findClassScopeStartIndex(lines=lines, startIndex=startIndex)
 
     def _findFunctionScopeEndIndex(self, lines: List[str], scopeStartIndex: int) -> int:
-        return self._findClassScopeEndIndex(lines=lines, scopeStartIndex=scopeStartIndex)
+        scopeStack = []
+        line = lines[scopeStartIndex].strip()
+        # For a single line function, return the `scopeStartIndex` directly
+        if self.__syntaxMatcher.matchFunctioinScopeEnd(line) or \
+                self.__syntaxMatcher.matchAbstractFunctioinScopeEnd(line):
+            return scopeStartIndex
```

Comment:
Should this be `and` instead of `or`?

Reply:
The function scope end and abstract function scope end are not the same in most of languages. I think it should be `or` instead.

Reply:
:+1: I misread the code actually!